### ISSUE TITLE
Node Sass does not yet support your current environment

### ISSUE THREAD

**Issue by bennycode** [knksmith57/hexo-renderer-sass#42]
I am getting the following error:

```
ERROR {
  err: Error: Node Sass does not yet support your current environment: Windows 64-bit with Unsupported runtime (93)
  For more information on which environments are supported please see:
  https://github.com/sass/node-sass/releases/tag/v4.14.1
      at module.exports (C:\dev\projects\bennycode\typescript.tv\node_modules\node-sass\lib\binding.js:13:13)
      at Object.<anonymous> (C:\dev\projects\bennycode\typescript.tv\node_modules\node-sass\lib\index.js:14:35)
      at Module._compile (node:internal/modules/cjs/loader:1108:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1137:10)
      at Module.load (node:internal/modules/cjs/loader:988:32)
      at Function.Module._load (node:internal/modules/cjs/loader:828:14)
      at Module.require (node:internal/modules/cjs/loader:1012:19)
      at require (node:internal/modules/cjs/helpers:93:18)
      at Object.<anonymous> (C:\dev\projects\bennycode\typescript.tv\node_modules\hexo-renderer-scss\lib\renderer.js:1:14)
      at Module._compile (node:internal/modules/cjs/loader:1108:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1137:10)
      at Module.load (node:internal/modules/cjs/loader:988:32)
      at Function.Module._load (node:internal/modules/cjs/loader:828:14)
      at Module.require (node:internal/modules/cjs/loader:1012:19)
      at req (C:\dev\projects\bennycode\typescript.tv\node_modules\hexo\lib\hexo\index.js:292:23)
      at C:\dev\projects\bennycode\typescript.tv\node_modules\hexo-renderer-scss\index.js:1:81
} Plugin load failed: %s hexo-renderer-scss
```

I am using: Win 10 Pro with Node v16.0.0

**Comment by huuhuybn** [comment]
hello, you fixed it yet ?

**Comment by JLHwung** [comment]
This issue is fixed in #46.
